cudamapper/src/overlapper_triggered.cu

```diff
@@ -24,11 +24,6 @@
 
 #include <claraparabricks/genomeworks/utils/cudautils.hpp>
 
-#ifndef NDEBUG // only needed to check if input is sorted in assert
```

Comment:
Does this still compile in debug mode? I think lines 255/260 require these headers

Reply:
Fixed